utils/plot.hpp

```diff
@@ -60,8 +60,8 @@ int scatter(const std::string& fname,
 
   // String object representing the name of the feature to be used to mask the plot data points.
   PyObject* pMaskCol = PyUnicode_FromString(maskCol.c_str());
-  PyTuple_SetItem(pArgs, 4, pMaskCol);
-
+  PyTuple_SetItem(pArgs, 4, pMaskCol);  
+  
```

Comment:
```suggestion
  PyTuple_SetItem(pArgs, 4, pMaskCol);

```

Remove some extra spaces.